Kernel/Time/TimeManagement.cpp

```diff
@@ -186,14 +187,14 @@ UNMAP_AFTER_INIT void TimeManagement::initialize([[maybe_unused]] u32 cpu)
             // would trigger a deadlock trying to get the s_the instance while
             // creating it.
             if (auto* apic_timer = APIC::the().initialize_timers(*s_the->m_system_timer)) {
-                dmesgln("Time: Using APIC timer as system timer");
+                dmesgln("Duration: Using APIC timer as system timer");
```

Comment:
errr… those messages should not really be changed (those refer more to the kernel “module”). 🤔

Reply:
I'm not sure what this means or what that has to do with my time changes.